src/features/advanced/cxpo-auto-pricing.ts

```
@@ -0,0 +1,200 @@
+
+    const orderInfo = computed(() => cxobStore.getByTicker(tile.parameter!));
+    const currencyCode = orderInfo.value?.currency.code ?? '';
+
+    const effectivePriceLabel = await $(form.children[9], C.StaticInput.static);
+    effectivePriceLabel.style.display = 'none';
+    const effectivePriceLabelParent = effectivePriceLabel.parentElement!;
+    const featureEffectivePriceLabels = createDualLabels(effectivePriceLabelParent, currencyCode);
+
+    const volumeLabel = await $(form.children[10], C.StaticInput.static);
+    volumeLabel.style.display = 'none';
+    const volumeLabelParent = volumeLabel.parentElement!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)